lib/building.rb

```diff
@@ -0,0 +1,17 @@
+class Building
+  attr_reader :units,
+              :renters
+
+  def initialize
+    @units = []
+    @renters = []
+  end
+
+  def add_unit(unit_instance)
+    @units << unit_instance
+  end
+
+  def add_renter(renter_instance)
+    @renters << renter_instance.name
+  end
```

Comment:
We don't add renters to a building, they are attached to units. Renters should go through all units and collect the renters.